CCDB/src/CCDBLinkDef.h

```diff
@@ -33,5 +33,9 @@
 #pragma link C++ class o2::CDB::GridStorageFactory+;
 #pragma link C++ class o2::CDB::GridStorageParameters+;
 #pragma link C++ class o2::CDB::XmlHandler+;
+/// for the unit test
+#pragma link C++ class TestClass+;
+#pragma link C++ class o2::TObjectWrapper<TestClass>+;
```

Comment:
Does this mean that for every object also the instantiation of the wrapper with this object needs to be added?

Reply:
Unfortunately yes. In this particular case this might happen automatically via a tool (clang-tiday) because we can realize at compile when a class is written as a condition. In any case, this was the quickest path to allow writing any class to CCDB. The other option of modifying the Condition API would take longer but is not excluded.